docs/oauth2.md

```diff
@@ -229,6 +171,50 @@ auth_oauth2.algorithms.1 = HS256
 auth_oauth2.algorithms.2 = RS256
 ```
 
+Each `auth_oauth2.resource_servers.<id/index>.` entry has the following settings:
+
+* `id`
+* `resource_server_type`
+* `additional_scopes_key`  
+* `scope_prefix`  
+* `preferred_username_claims`
+* `oauth_provider_id`
+
+Typically, a numeric value is used as `index`, e.g.  `auth_oauth2.resource_servers.1.id = rabbit_prod`. However, it can be any string, e.g. `auth_oauth2.resource_servers.rabbit_prod.jwks_url = http://some_url`. By default, the `index` is the resource server's id. However, you can override it via the `id` attribute like in `auth_oauth2.resource_servers.1.id = rabbit_prod`.
+
+Here is an example which configures two resources (`prod` and `dev`) which are used by the users and clients managed by
+the same Identity Provider whose issuer url is `https://my-idp.com/`:
+
+```ini
+auth_oauth2.issuer = https://my-idp.com/
+auth_oauth2.resource_servers.1.id = prod
+auth_oauth2.resource_servers.2.id = dev
+```
+
+Each `auth_oauth2.oauth_providers.<id/index>.` entry has the following settings:
+
+* `issuer`
+* `token_endpoint`
+* `jwks_uri`
+* `https.cacertfile`
+* `https.depth`
+* `https.verify`
+* `https.fail_if_no_peer_cert`
+* `https.hostname_verification`
+
+
+Here is an example which configures two resources (`prod` and `dev`) where each resource is managed by two distinct Identity Provider:
+
```

Comment:
should "Identity Provider" be plural, you mention "two" above also does it need to be in caps "identity providers"